Doc/library/zipfile.rst

```diff
@@ -637,22 +641,41 @@ file:
    .. versionadded:: 3.8
       The *strict_timestamps* keyword-only argument
 
+   .. versionadded:: 3.9
+      The *follow_symlinks* keyword-only argument
+
 
 Instances have the following methods and attributes:
 
 .. method:: ZipInfo.is_dir()
 
-   Return ``True`` if this archive member is a directory.
+   Return :const:`True` if this archive member is a directory.
 
+   .. versionadded:: 3.6
    This uses the entry's name: directories should always end with ``/``.
 
-   .. versionadded:: 3.6
+   .. versionchanged:: 3.9
+   Use the filemode contained in external file attributes.
```

Comment:
Which method/attribute does this line cover?

Reply:
`ZipInfo.is_dir()`, which uses the filemode with this PR, instead of checking if the name finishes with a "/"

Reply:
I see. Probably good to keep both; maybe to keep the wording consistent you could do:

```suggestion
  This uses the filemode contained in external file attributes.
```

Reply:
change made in ca5e302